src/sentry/static/sentry/app/components/button.jsx

```diff
@@ -44,6 +44,10 @@ class Button extends React.Component {
      * `children` will be used by default (only if it is a string), but this property takes priority.
      */
     label: PropTypes.string,
+    /**
+     * Passed down to built-in tooltip component
+     */
+    containerDisplayMode: PropTypes.string,
```

Comment:
I think a more generic `tooltipOptions` would be better since it would describe the prop more (esp in context of `<Button>` where you would assume it would affect Button somehow).